content/articles/broadcasting-a-pirate-fm-radio-station-using-a-raspberry-pi/index.md

```diff
@@ -0,0 +1,79 @@
+Setting up a private radio station is easy using a Raspberry Pi. This process should take thirty to sixty minutes to complete. By the end of this hour, you should be able to broadcast within a radius of around fifty meters. However, this is also facilitated by the use of a good antenna. 
+<div style="text-align: center; padding-bottom: 5%;">
+<img style="padding: 0; margin:0;" src="/engineering-education/broadcasting-a-pirate-fm-radio-station-using-a-raspberry-pi/raspberry21.jpg" alt="Raspberry Pi diagram"><br>
+<span style="font-size: 11px;">Figure 1: Sample Rapberry Pi <a href="https://unsplash.com/photos/raLeFIxXgDY">Raspberry Pi</a>, 2021.<span>
+</div>
```

Comment:
Here, I'd like you to be creative and put a picture of a raspberry pi with connections coming out of the GPIO pins and the antenna connected to it. You don't have to get it through Unsplash. You could build one yourself (which i assume you have) and take a photo of it. That would really grab the readers attention.  
Secondly, the format you've used there is wrong. Please use the format below:

For example, yours should be:

![Raspberry Pi diagram](/engineering-education/broadcasting-a-pirate-fm-radio-station-using-a-raspberry-pi/raspberry21.jpg)

*[Image Source: Unsplash](https://unsplash.com/photos/raLeFIxXgDY)*